components/password_reset_form/password_reset_form.js

```diff
@@ -92,11 +95,11 @@ export default class PasswordResetForm extends React.PureComponent {
                         </p>
                         <div className={formClass}>
                             <LocalizedInput
-                                ref={this.passwordInput}
```

Comment:
do we have to revert it to old-style references? If it is safer, I'm ok with that.

Reply:
Good catch. That shouldn't be happening since the non-reverted version of LocalizedInput doesn't work with string refs